src/tests/time-sheets.test.js

```diff
@@ -1,5 +1,160 @@
-describe('example', () => {
-  it('Test', () => {
-    expect(true).toEqual(true);
+import request from 'supertest';
+import mongoose from 'mongoose';
+import app from '../app';
+import TimeSheets from '../models/Time-sheets';
+import Employees from '../models/Employees';
+import Projects from '../models/Projects';
+import Tasks from '../models/Tasks';
+import timeSheetsSeed from '../seeds/time-sheets';
+import employeesSeed from '../seeds/employees';
+import projectsSeed from '../seeds/projects';
+import tasksSeed from '../seeds/tasks';
+
+describe('Time-sheet - Unit tests', () => {
+  beforeAll(async () => {
+    await TimeSheets.collection.insertMany(timeSheetsSeed);
+    await Employees.collection.insertMany(employeesSeed);
+    await Projects.collection.insertMany(projectsSeed);
+    await Tasks.collection.insertMany(tasksSeed);
+  });
+
+  describe('GET /time-sheets', () => {
+    test('should return status code 200', async () => {
+      const response = await request(app).get('/time-sheets').send();
+      expect(response.status).toBe(200);
+      expect(response.body.error).toBeFalsy();
+      expect(response.body.data).toBeDefined();
+      expect(response.body.message).toBe('Time sheets found');
+    });
+  });
+
+  describe('GET BY ID /:id', () => {
+    test('should return status code 200', async () => {
+      const response = await request(app).get('/time-sheets/63535ae7fc13ae517a000031').send();
+      expect(response.status).toBe(200);
+      expect(response.body.error).toBeFalsy();
+      expect(response.body.data).toBeDefined();
+      expect(response.body.message).toBe('Time sheet found');
+    });
+  });
+
+  describe('GET BY ID ERROR /:id', () => {
+    test('should return status code 400', async () => {
```

Comment:
Why the request is returning a status 40?
You have to add the purpose why this is going to return an error.
```suggestion
    test('should return status code 400 when the id is invalid', async () => {
```